### PR TITLE
samples: wifi: radio_test: Correct command output and order

### PR DESCRIPTION
**Pull request by prsi98**
[SHEL-3047]: Update command output, subcommand order and argument.

[SHEL-3047]: https://nordicsemi.atlassian.net/browse/SHEL-3047?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by srkanordic — APPROVED** [review]

**Review by bama-nordic — APPROVED** [review]

**Review by richabp — COMMENTED** [review]
Is it required to mention this update in the Changelog?

**Comment by prsi98** [comment]
> Is it required to mention this update in the Changelog?

added changelog.

File: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst [hunk 1]
```diff
@@ -520,6 +520,7 @@ Wi-Fi samples
 * :ref:`wifi_radio_test` sample:
 
   * Added capture timeout as a parameter for packet capture.
+  * Increased subcommand scope and corrected subcommand behavior.
```
Comment (richabp) [inline doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst:523]: Can it be rephrased as follows?
"Expanded the scope of subcommand and rectified its behavior."

Comment (prsi98) [reply doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst:523]: Sure, but I'm addressing two different subcommands here. 

_wifi_radio_test show_config_ Increased its scope

_wifi_radio_test tx_pkt_preamble_ rectified its behavior.

Comment (richabp) [reply doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst:523]: Ok, then I would suggest to mention the subcommands as well.

Comment (richabp) [reply doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst:523]: Something like this?

Expanded the scope of ``wifi_radio_test show_config`` subcommand and rectified the behavior of ``wifi_radio_test tx_pkt_preamble`` subcommand.

**Review by richabp — APPROVED** [review]

**Review by sachinthegreen — APPROVED** [review]

**Pull request merged**